tools/bbc_config_helper.py

```
@@ -0,0 +1,48 @@
+"""BBC 配置处理工具"""
+from pathlib import Path
+import shutil
+
+
+def prepare_bbc_team_config(assets_options_dir: Path) -> None:
+    """
+    准备 BBC 队伍配置文件
+    
+    在源目录中：
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)